dashboard/config/scripts/levels/VPL - CSD Module 5 - Differentiation_2022.level

```diff
@@ -5,7 +5,7 @@
+    "long_instructions": "# Differentiation\r\n\r\n**Estimated Time: 10 minutes**\r\n\r\n## Overview\r\n\r\nDuring the summer we introduced the following definition for **inclusion** adapted from the <a href=\"https://drive.google.com/file/d/1NNS1M4aY4WpfJ8glZBUqIDPvhAimZRtr/view\" target=\"_blank\">“Guide to inclusive computer science education”</a> education” compiled by Microsoft, in partnership with CSTA, Code.org, NCWIT and other organizations: \r\n\r\n*“Creating learning environments that are accessible and welcoming of students’ identities, backgrounds, differences and perspectives without barriers or judgment.  This means actively attending to gender, race,  ethnicity, ability or socioeconomic status.”*\r\n\r\n## Goals\r\n\r\n* Identify teacher support available within lessons and the curriculum that support differentiation to promote inclusion\r\n* Identify current classroom strategies you use that meet various needs of learners and promote inclusion\r\n\r\n## Tasks\r\n\r\n* Review the lesson plan for the model lesson, <a href=\"https://studio.code.org/s/csd5-2022/lessons/5\" target=\"_blank\">Lesson 5:  Representing Numbers</a>\r\n* Preview the <a href=\"https://docs.google.com/document/d/15Y6Mo3fMUtHq16SSskABgcYE-5xfk-Ewv0z_o4uaLdI/edit\" target=\"_blank\">CSD Guide to Differentiation 2021</a>\r\n\r\n## Reflection\r\n\r\n**Prompts:**\r\n* What teacher supports and strategies are available within Lesson 5 (e.g. Teacher Tips, etc) and the CSD Guide to Differentiation 2021?\r\n* What strategies do you currently use in your classroom that promote inclusion and meet the unique learning needs of your students? \r\n\r\n\r\n",
     "allow_user_uploads": "false",
```

Comment:
Not sure what changed here. Can you help me?

Reply:
<img width="432" alt="Screen Shot 2022-06-28 at 6 46 26 PM" src="https://user-images.githubusercontent.com/8001765/176333443-938f4d6e-f797-41c5-b6b4-c1bae75a5927.png">

Reply:
csd5-2021 --> csd5-2022